src/containers/header/index.js

```diff
@@ -4,26 +4,43 @@ import { NavLink } from 'react-router-dom';
 
 import PropTypes from 'prop-types';
 import { MdSearch } from 'react-icons/md';
+import { FaBars, FaWindowClose } from 'react-icons/fa';
 
 import { logOutRequest } from '../../actions/auth';
-import Burger from './burgerMenu';
 import Login from '../auth/login';
 import Register from '../auth/register';
 import ForgotPassword from '../auth/forgotPassword';
 
 import styles from './styles.module.scss';
+import './menu.scss';
 
 const Header = props => {
   const [width, setWidth] = useState(window.innerWidth);
   const [isOpenLog, setModalStatusLog] = useState(false);
   const [isOpenReg, setModalStatusReg] = useState(false);
+  const [isOpenMenu, setMenuStatus] = useState(false);
   const [isOpenForgotPass, setModalForgotPass] = useState(false);
 
   useEffect(() => {
     const handleWidth = () => setWidth(window.innerWidth);
     window.addEventListener('resize', handleWidth);
   }, []);
 
+  useEffect(() => {
+    if (width > 768) {
+      const toggleMenu = document.querySelector('#menu');
+      toggleMenu.classList.remove('mobileMenuActive');
+      setMenuStatus(false);
+    }
+  });
+
+  const toggleMenuFunc = () => {
+    if (width < 768) {
+      const toggleMenu = document.querySelector('#menu');
+      toggleMenu.classList.toggle('mobileMenuActive');
+    }
+  };
+
```

Comment:
Remove ctrl+c ctrl+v code

Reply:
I think he needs just to push `const toggleMenu = document.querySelector('#menu');`
from the functions.